templates/OME-TIFF Conversion.ipynb

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
<p>Okay sorry to be super anal about this, but can we name this "OME-TIFF to single-channel TIFFs"? FOV stands for field-of-view which is confusing since OME-TIFFs are also capturing a FOV.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>